src/sentry/static/sentry/app/views/performance/data.tsx

```diff
@@ -141,3 +146,56 @@ export function generatePerformanceEventView(
 
   return EventView.fromNewQueryWithLocation(savedQuery, location);
 }
+
+export function generatePerformanceVitalDetailView(
+  organization: LightWeightOrganization,
+  location: Location
+): EventView {
+  const {query} = location;
+
+  const vitalName = vitalNameFromLocation(location);
+
+  const keyTransactionsFeature = organization.features.includes('key-transactions');
+  const keyTransactionFields = keyTransactionsFeature ? ['key_transaction'] : [];
+
+  const hasStartAndEnd = query.start && query.end;
+  const savedQuery: NewQuery = {
+    id: undefined,
+    name: t('Performance'),
+    query: 'event.type:transaction',
+    projects: [],
```

Comment:
Should you copy the projects from the current view?

Reply:
Projects is already getting picked up in the Eventview function (double checked that this correctly provides projects from landing to vital detail)